src/main/java/seedu/clinic/storage/UserMacrosStorage.java

```diff
@@ -0,0 +1,44 @@
+package seedu.clinic.storage;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import seedu.clinic.commons.exceptions.DataConversionException;
+import seedu.clinic.model.ReadOnlyUserMacros;
+import seedu.clinic.model.UserMacros;
+
+/**
+ * Represents a storage for {@link UserMacros}.
+ */
+public interface UserMacrosStorage {
+    /**
+     * Returns the file path of the UserMacros data file.
+     */
+    Path getUserMacrosFilePath();
+
+    /**
+     * Returns user macro data as a {@link ReadOnlyUserMacros}.
+     *   Returns {@code Optional.empty()} if storage file is not found.
```

Comment:
Some extra spaces here can be removed as well

Reply:
Thanks! will fix it.